example_config.toml

```diff
@@ -50,3 +50,12 @@ max_open_announces = 50
 # a connection is eligible for forced pruning
 # when the max socket limit is reached
 prune_timeout = 15
+
+[ip_filter]
+# 1 - block prefix
```

Comment:
Why not 0 to block, allow for anything else. Since other values specify priority let’s leave a comment around this too.

Reply:
Feel free to correct my comments, because my english isn't perfect.
Also i haven't found ip priority queue in code base.